idaes_examples/notebooks/docs/unit_models/custom_unit_models/LiqLiqExtractor.py

```diff
@@ -0,0 +1,62 @@
+import pyomo.environ as pyo
+import idaes.core
+import idaes.models.unit_models
+from idaes.core.solvers import get_solver
+import idaes.logger as idaeslog
+from pyomo.network import Arc
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.initialization import InitializationStatus
+from idaes.core.initialization.block_triangularization import (
+    BlockTriangularizationInitializer,
+)
+from Org_property import OrgPhase
+from Aq_property import AqPhase
+from liquid_extraction import LiqExtraction
+
+m = pyo.ConcreteModel()
+m.fs = idaes.core.FlowsheetBlock(dynamic=False)
+m.fs.org_properties = OrgPhase()
+m.fs.aq_properties = AqPhase()
+
+m.fs.lex = LiqExtraction(
+    dynamic=False,
+    has_pressure_change=False,
+    organic_property_package=m.fs.org_properties,
+    aqueous_property_package=m.fs.aq_properties,
+)
+m.fs.lex.organic_inlet.flow_vol.fix(80 * pyo.units.L / pyo.units.hour)
+m.fs.lex.organic_inlet.temperature.fix(300 * pyo.units.K)
+m.fs.lex.organic_inlet.pressure.fix(1 * pyo.units.atm)
+m.fs.lex.organic_inlet.conc_mass_comp[0, "NaCl"].fix(0 * pyo.units.g / pyo.units.L)
```

Comment:
You should avoid setting any concentration or mole/mass fraction to 0 as it can cause issues with more rigorous property calculations. Whilst it probably works here, I think it is good practice to learn to avoid doing this at all times (and as an example show users not to do it too).